tsconfig.json

```diff
@@ -1,41 +1,18 @@
 {
+    "extends": "@apify/tsconfig",
     "compilerOptions": {
-        "target": "es2018",
-        "module": "commonjs",
-        "moduleResolution": "node",
+        "target": "ES2020",
         "lib": [
```

Comment:
Do you *need* to provide lib still? (I don't remember if I made lib include dom and es2020 in the module, I'll check in a few, but if I didn't include `dom`, feel free to ignore this) And target...can't we just move that in the tsconfig module, since all modules should probably be 15+

Reply:
The base config targets `es2019` (I guess we can change that in the base one to `es2020`), also I see it uses `esnext` in the lib, not sure how safe is that. We need `dom` lib here, I don't think it should be part of the base config. The lib `es2020` is based on [this](https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping#node-14).

Reply:
We can update the target in the module to es2020 then, and keep the rest as is 👍